data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-creation-v1.0.xml

```diff
@@ -217,4 +191,6 @@
                        constraintName="utxo_transaction_cpk_pkey"
                        tableName="utxo_transaction_cpk"/>
     </changeSet>
+
+    <include file="net/corda/db/schema/vnode-vault/migration/ledger-utxo-relevant-transaction-state-creation-v1.0.xml"/>
```

Comment:
Good idea to split this out into a separate file!

Reply:
I don't think it makes technically a difference, but wouldn't it be more consistent to have this include in the `db.changelog-master.xml` file?
Where it is here may get a bit confusing when a new version is introduced.